device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/TransportClientTest.java

```diff
@@ -320,7 +320,7 @@ public void registerDeviceClientThrowsDeviceClientNull()
         TransportClient transportClient = new TransportClient(iotHubClientProtocol);
 
         // act
-        Deencapsulation.invoke(transportClient, "registerDeviceClient", (DeviceClient)null);
```

Comment:
You don't need to make this change, my branch should have this fix already

Reply:
I know ! I have rebased onto your branch before submitting the PR, no idea why it shows up in the PR, if your branch already has this change it should just ignore it in the PR ... strange

Reply:
I'd pull from my branch again. I've done some force pushes after squashing these kinds of commits